testing/resources/renv_hooks_repo/.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,48 @@
+# parsing file
+-   id: parse-file-no-opts-no-args
+    name: Say hi
+    entry: Rscript parse-file-no-opts-no-args.R
+    language: renv
+    files: \.[Rr]$
```

Comment:
we should probably add `r` as a language to pre-commit/identify if it's not there already

Reply:
Apparently it is there already: 
```python
identify.tags_from_path('testing/resources/renv_hooks_repo/hello-world.R')
#> {'r', 'non-executable', 'text', 'file'}
```
So I can use this in that unit test. However, there are other R file types we want to support in some way in `identify` probably: `.Rprofile`, `.Rmd`, `.Rmarkdown` and `Rscript` shebang. I think I'd like to think about that later, and check with https://github.com/lorenzwalthert/precommit which ones could be useful.

Reply:
cool cool -- the place to do that is in https://github.com/pre-commit/identify there's an `extensions.py` file where you can add those!